packages/did-core-test-server/suites/did-spec/utils.js

```diff
@@ -18,7 +18,14 @@ const isValidURL = (data) => {
   return valid;
 };
 
+const isValidDID = (did) => {
+  const didRegex = /did:(?<method>[a-z0-9]+):(?<idchar>[a-zA-Z0-9:\\-_]+)/;
```

Comment:
Feels like there is going to be a ton of duplication w/ this test suite and the Transmute one: https://github.com/transmute-industries/did-core/blob/main/packages/did-core-tests/json-schemas/did.json

Are there any plans to merge these two things?

Reply:
I can published the package and we can use it as a dependency here... but I'm not going to try to maintain our test suite in a W3C repo governed by W3C Consensus :)

Reply:
> I'm not going to try to maintain our test suite in a W3C repo governed by W3C Consensus

What could possibly go wrong? :P 

Don't worry about it -- I expect we'll be able to get through it w/o the JSON Schema stuff... but would have probably made the tests easier/nicer than just dumping stuff in `util.js`.

Reply:
I'm not too fond of duplication 😝    I think having a common library in this package is a good thing. Anyone who doesn't want to depends on this package can create a similar function in their package.